pkgs/top-level/python-packages.nix

```diff
@@ -12241,6 +12241,10 @@ self: super: with self; {
 
   pyowm = callPackage ../development/python-modules/pyowm { };
 
+  pyoxigraph = callPackage ../development/python-modules/pyoxigraph {
+    inherit (darwin.apple_sdk.frameworks) IOKit Security;
```

Comment:
```suggestion
    inherit (pkgs.darwin.apple_sdk.frameworks) IOKit Security;
```

Reply:
With this change I was able to build the following packages on aarch64-darwin:
- python39Packages.pyoxigraph
- python310Packages.pyoxigraph
- python311Packages.pyoxigraph
- python312Packages.pyoxigraph
- python313Packages.pyoxigraph